OWNERS

```diff
@@ -0,0 +1,25 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- david-martin
+- mikenairn
+- philbrookes
+- sergioifg94
+- R-Lawton
+- roivaz
+- pmccarthy
+- maleck13
+- makslion
+- jasonmadigan
+
+reviewers:
+- david-martin
+- mikenairn
+- philbrookes
+- sergioifg94
+- R-Lawton
+- roivaz
+- pmccarthy
+- maleck13
+- makslion
+- jasonmadigan
```

Comment:
@maleck13 Any opinions on what this should look like? Approvers are basically able to merge PRs

Reply:
Lets go with 

```
- david-martin
- mikenairn
- philbrookes
- sergioifg94
- maleck13
```

initially we can add others as knowledge grows etc

Reply:
As it stands right now all those people have the ability to hit the "merge" button :thinking: If we narrow the `approvers` list we should, probably, review what people can do in this repo. Reduced the list to people mentioned above